### DISCUSSION TITLE
Minimum Disk Capacity for Minio Distributed mode

### DISCUSSION THREAD

**Discussion by bh-ashishsharma** [minio/minio#15027] [Q&A] [answered]
Hello, 

I am setting up my minio cluster in distributed mode with 4 nodes with 1 attached drive to each node.
I am currently doing it for lower environments. 
For my current usage for dev/testing environments, I am okay with 200 GB drive each (200 * 4 = 800 GB) which would give me 400 GB drive space for use.

I wanted to know if there is any minimum capacity requirement for each drive? I got confused because on the [erasure-code-calculator](https://min.io/product/erasure-code-calculator) it's not allowing me to set the drive capacity less than 1 TB

**Comment by klauspost** [comment, marked as answer]
There is no minimum requirement. There is rarely a need for such a small install, so we deal with TBs.

You can just enter 200TB disks, the math will be the same.
